packages/is/plain-object.js

```diff
@@ -1,4 +1,7 @@
-module.exports = function isPlainObject(value) {
-  return Boolean(value) && typeof value === 'object' &&
-    Object.getPrototypeOf(value) === Object.prototype;
-};
+'use strict';
+
+const isPlainObject = (value) =>
+  // ??? purpose of Boolean(value)
```

Comment:
Try
```js
isPlainObject(null)
```

With `Boolean` it returns `false`. Without `Boolean` it returns `null`.

Reply:
Ok. My questioning was originally: is `Boolean(value)` equivalent to `!!value`.
I tend to forget the "typeof null" thing, sorry.